binder/environment.yml

```diff
@@ -1,10 +1,10 @@
 name: phenomenal
 channels:
   - openalea
-  - defaults
   - conda-forge
+  - defaults
```

Comment:
Can we just depends on conda-forge?
I think either openalea or defaults are not needed.